install-template.sh

```diff
@@ -1,25 +1,25 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 
 # This is the install script for '@@toolName@@'.
 
-set -euo pipefail
+set -eu
 
 test_nix_installation () {
-  which nix > /dev/null &&
-  nix --version > /dev/null &&
-  true
+  which nix > /dev/null
+  nix --version > /dev/null
 }
 
 install_nix () {
-  tmp=$(mktemp -d)
-  echo "extra-substituters = https://cache.garnix.io" >> $tmp/nix-extra-config
-  echo "extra-trusted-public-keys = cache.garnix.io:CTFPyKSLcx5RMJKfLo5EEPUObbA78b0YQ2DTCJXqr9g=" >> $tmp/nix-extra-config
+  TMP=$(mktemp -d)
+  echo "extra-substituters = https://cache.garnix.io" >> "$TMP/nix-extra-config"
+  echo "extra-trusted-public-keys = cache.garnix.io:CTFPyKSLcx5RMJKfLo5EEPUObbA78b0YQ2DTCJXqr9g=" >> "$TMP/nix-extra-config"
 
-  curl -L https://nixos.org/nix/install -o $tmp/install.sh
-  chmod u+x $tmp/install.sh
+  curl -L https://releases.nixos.org/nix/nix-2.17.1/install -o "$TMP/install.sh"
+  chmod u+x "$TMP/install.sh"
 
-  $tmp/install.sh --nix-extra-conf-file $tmp/nix-extra-config --daemon --yes
+  "$TMP/install.sh" --nix-extra-conf-file "$TMP/nix-extra-config" --daemon --yes
 
+  # shellcheck source=/dev/null
```

Comment:
Should we remove this?

Reply:
This is a comment to tell shellcheck to not complain about sourcing a file that it cannot check. (And in the nix build sandbox where we run `shellcheck` this file is not available.) See also https://www.shellcheck.net/wiki/SC1091.